.github/workflows/devbuild.yml

```diff
@@ -406,6 +406,9 @@ jobs:
 
       - name: cmake run_viewer_pytest
         run: |
+          $pyside6_path = $(python3 -c "import sys, os, PySide6; sys.stdout.write(os.path.dirname(PySide6.__file__))")
```

Comment:
I am guess that you add the environment variable only in this step because you do not want to interfere the non-view step run_gtest?  If so, I think we can use a comment to spell out the subtle intention.  Otherwise maintainers will wonder why not simply pushing the environment PATH to the global when pyside6 is installed.

Reply:
No, I have tried to add the environment in a new step, but it seems didn’t work, so I put these code in this step.
I will find out why and use other PR to add comment to explain about this!

Reply:
No problem.